### PR TITLE
Fix project options when list is toggled/Fix broken pagination of projects in navigator

### PR DESCRIPTION
**Pull request by caseychoiniere**

### CODE REVIEW

File: app/scripts/stores/mainStore.js [hunk 1]
```diff
@@ -328,10 +328,8 @@ export class MainStore {
             let headers = json[1].map;
             if(page <= 1) {
                 this.projects = results;
-                this.listItems = results;
             } else {
                 this.projects = [...this.projects, ...results];
-                this.listItems = [...this.listItems, ...results];
```
Comment (ChetCorey) [inline app/scripts/stores/mainStore.js]: Will this break ListItems in `{dds}/projects/{:id}` and `{dds}/folders/{:id}`

Comment (caseychoiniere) [reply app/scripts/stores/mainStore.js]: No. listItems should never be set in getProjects. They are separate concerns. I think this is something you added at some point and forgot to get rid of. It's not in ua_test or prod.

**Review by ChetCorey — APPROVED** [review]
Looks good

**Review by ChetCorey — APPROVED** [review]
Looks good 👍

**Pull request merged**